inc/Engine/WPRocketUninstall.php

```diff
@@ -180,6 +197,11 @@ private function delete_plugin_data() {
 		// Delete all user meta related to WP Rocket.
 		delete_metadata( 'user', '', 'rocket_boxes', '', true );
 
+		// Delete all post meta related to WP Rocket.
+		foreach ( $this->wpr_post_meta as $pm ) {
+			delete_metadata( 'post', '', $pm, '', true );
+		}
```

Comment:
```suggestion
		foreach ( $this->post_meta as $post_meta ) {
			delete_post_meta_by_key( "_rocket_exclude_{$post_meta}" );
		}
```

We should avoid too short or unexplicity names for variables, so I suggest using `$post_meta` instead of `$pm` for readability.

`delete_post_meta_by_key()` is a wrapper function that achieves the same result, but it's easier to read